Mathlib/Condensed/Discrete/LocallyConstant.lean

```diff
@@ -221,31 +220,29 @@ variable (P) (X : TopCat.{max u w})
     (hs : ∀ ⦃X Y : CompHausLike P⦄ (f : X ⟶ Y), EffectiveEpi f → Function.Surjective f)
 
 /-- `locallyConstantIsoContinuousMap` is a natural isomorphism. -/
-noncomputable def functorToPresheavesIsoTopCatToSheafCompHausLike (X : Type (max u w)) :
-    functorToPresheaves.{u, w}.obj X ≅
-      ((topCatToSheafCompHausLike P hs).obj (TopCat.discrete.obj X)).val :=
+noncomputable def functorToPresheavesIso (X : Type (max u w)) :
+    functorToPresheaves.{u, w}.obj X ≅ ((TopCat.discrete.obj X).toSheafCompHausLike P hs).val :=
   NatIso.ofComponents (fun S ↦ locallyConstantIsoContinuousMap _ _)
 
 /-- `CompHausLike.LocallyConstant.functorToPresheaves` lands in sheaves. -/
 @[simps]
-def functor :
-    have := CompHausLike.preregular hs
+def functor : haveI := CompHausLike.preregular hs
```

Comment:
I am not sure about the coding-style here: I would indend `have` as it was previously. Otherwise, LGTM.